bin/adhoc-scripts/MSRuleCleanerStandalone.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import os
+import time
+import logging
+
+from pprint import pformat
+from itertools import izip
+
+from WMCore.MicroService.Unified.MSRuleCleaner import MSRuleCleaner
+from WMCore.Database.CMSCouch import Database as CouchDB
+
+FORMAT = "%(asctime)s:%(levelname)s:%(module)s:%(funcName)s(): %(message)s"
+logging.basicConfig(stream=sys.stdout, format=FORMAT, level=logging.DEBUG)
+logger = logging.getLogger()
+
+# Service config
+msConfig = {"enableRealMode": False,
```

Comment:
Don't forget this configuration will have to be updated as well.

Reply:
I'd rather make a set of a runtime arguments and this one to be part of them. Working on that  at this very moment.

Reply:
Done